src/components/Booking.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div>
+		<h2>Booking</h2>
+		<iframe
+			v-if="bookingFormUrl"
+			id="bookingForm"
+			src="bookingFormUrl"
```

Comment:
It seems to be taking your variable a bit to literally

![image](https://user-images.githubusercontent.com/19759293/162010756-c5b4bccf-6391-4777-a4a2-6744cb2c233d.png)

Reply:
Have you tested it? I think src="bookingformUrl" is refereeing to the variable

Reply:
Yes, that image is from source inspection in the browser

Reply:
My bad I forgot to write `:src`